tgui/packages/tgui/styles/base.scss

```diff
@@ -12,4 +12,15 @@ $color-bg-start: color.adjust(
   $color-bg, $lightness: $color-bg-grad-spread) !default;
 $color-bg-end: color.adjust(
   $color-bg, $lightness: -$color-bg-grad-spread) !default;
-$border-radius: 2px !default;
+
+$unit: 12px;
+$font-size: 1 * $unit !default;
+$border-radius: 0.16em !default;
+
+@function em($px) {
+  @return 1em * ($px / $unit);
+}
+
+@function rem($px) {
+  @return 1rem * ($px / $unit);
+}
```

Comment:
```suggestion
@function em($px) {
  @return 1em * calc($px / $unit);
}

@function rem($px) {
  @return 1rem * calc($px / $unit);
}
```

Reply:
Right I saw this warning as well. I forgot to address it.